templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -1,14 +1,20 @@
 # Mention a Team Member
 
-![Mention a Team Member](/static/images/help/mention_user.png)
+![@-mention a team member](/static/images/help/mention_user.png)
 
-When having a conversation in Zulip, it is sometimes necessary to call
-the attention of another member. You can alert ("**@mention**") other
-users with a particular stream-message from within the body of the
-message itself. Type `@` and then begin typing their email address or
-one of their names, and the system will offer you tab-completions. The
-completed `@mention` will look something like: `@**name**`, and it
-will appear as **@name** in your posted message.
+When having a conversation in Zulip, you may sometimes need to call
+the attention of another member. You can alert other
+users within a particular stream through using **@mention** in
+the **Compose Message** box. Type `@` followed by their email address or one of
+their names (first name or surname), and
+Zulip will offer you tab-completions. A completed `@mention` will look
+something like: `@**name**`, where **name** is the user you want to alert, and it
+will appear as @\*\*name\*\* in the markdown of your message.
+
+!!! tip ""
+    You cannnot add the **@mention** feature to message edits since the user you
+    want to mention may have already read the original message and the method to
+    notify them is not clear.
```

Comment:
(For most documentation, the phrase "the X feature" can usually be simplified to just "X" and it'll read better).  @synicalsyntax do you think this should be in our style guide?